src/app/currency-converter/page.tsx

```diff
@@ -0,0 +1,14 @@
+import Converter from '../../components/converter';
+
+export default function Home() {
+
+
+    return (
+        <div>
+            <h3 className="text-center text-xl">This is the currency converter section</h3>
+
+
+            <Converter />
+        </div>
+    )
+}
```

Comment:
Indentation could be done better.

Reply:
@ArinzeJeffrey-droid. I forgot to format it. My bad. Good morning.